canonical-web-and-design/vanilla-framework
# Side navigation examples

## Done

Adds more examples for side navigation

Fixes #2865

## QA

- Run `./run` or [demo]
- Clicking on side nav in these examples is possible (JS is now added to make them interactive without navigating to other pages)

Check updated side nav examples (with wrapping, truncating, icons, etc):
https://vanilla-framework-canonical-web-and-design-pr-2937.run.demo.haus/docs/examples/patterns/side-navigation/default

<img width="725" alt="Screenshot 2020-03-18 at 15 50 26" src="https://user-images.githubusercontent.com/83575/76973471-48dcd480-6930-11ea-8030-435e61ee9980.png">


MAAS current custom docs layout: https://vanilla-framework-canonical-web-and-design-pr-2937.run.demo.haus/docs/examples/templates/maas-docs

<img width="1249" alt="Screenshot 2020-03-18 at 13 49 03" src="https://user-images.githubusercontent.com/83575/76962428-82591400-691f-11ea-83b8-3623bd6a10a9.png">



MAAS docs in grid: https://vanilla-framework-canonical-web-and-design-pr-2937.run.demo.haus/docs/examples/templates/maas-docs-grid

<img width="1250" alt="Screenshot 2020-03-18 at 13 49 17" src="https://user-images.githubusercontent.com/83575/76962433-85540480-691f-11ea-9953-6e522b4a525a.png">


Snapcraft publicise: https://vanilla-framework-canonical-web-and-design-pr-2937.run.demo.haus/docs/examples/templates/snapcraft-publicise

<img width="1249" alt="Screenshot 2020-03-18 at 13 49 35" src="https://user-images.githubusercontent.com/83575/76962435-86853180-691f-11ea-88c8-81a8a6a71b54.png">
